### ISSUE TITLE
Intermittent Error with Plugin.

### ISSUE THREAD

**Issue by HauntedEri** [easly1989/ffxiv_act_dfassist#67]
Hello! I've installed DFAssist according to the instructions and sometimes it works just fine, shows the instance that popped in the ACT notifications. Other times however, it does not show and instead throws this error at me:

> A: Error while analyzing Message: Controls created on one thread cannot be parented to a control on a different thread.

I'm assuming this is probably user error on my end somehow, but I do not have the slightest as to how to fix it. Google didn't give me anything, so I'm asking here now.

**Comment by zman1719** [comment]
I have the same issue.  I've tried everything and cannot get this plugin to work.  I've tried uninstalling all other plugins but this one and the FFXIV parser, reinstalling ACT altogether, running with no firewall, running game in Windowed, Borderless and Full Screen and more.  Nothing I do lets me get around this error.  I'm using Windows 10 with the correct .NET framework installed as well.  No other plugin in ACT has an issue, just this one.
